.github/workflows/unit_tests.yml

```diff
@@ -23,7 +23,12 @@ jobs:
       run: |
         go get gotest.tools/gotestsum@v0.4.2
         # TODO: validate bin/protoc-go.sh does not dirty the repo
-        gotestsum -- -cover -race -v -mod=readonly ./...
+        gotestsum -- -cover -coverprofile=coverage.out -race -v -mod=readonly ./...
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        files: ./coverage.out
+        flags: unittests,golang
```

Comment:
The report shows:
```

Flag | Coverage Δ | &nbsp;
-- | -- | --
golang | 42.44% &lt;0.00%&gt; (?) | &nbsp;
javascript | 66.69% &lt;0.00%&gt; (?) | &nbsp;
unittests | 44.37% &lt;0.00%&gt; (?)
```
These are all unit tests, so I think the `unittests` flag is redundant.